internal/generator/vector/output/elasticsearch/elasticsearch.go

```diff
@@ -248,3 +249,15 @@ func BasicAuth(id string, o logging.OutputSpec, secret *corev1.Secret) []Element
 
 	return conf
 }
+
+func safePath(path string) string {
+	if !strings.Contains(path, "-") {
```

Comment:
We can just make it always use "get" .  It should work for both cases

Reply:
yes., it will work for sure